test/langtools/tools/javac/processing/model/util/types/TestInvalidInputs.java

```diff
@@ -161,20 +160,21 @@ void testUnboxedType() {
     void testGetWildcardType() {
         // Reference types are ArrayType, DeclaredType, ErrorType, NullType, TypeVariable
         // non-reference: ExecutableType, IntersectionType, NoType, PrimitiveType, UnionType, WildcardType
-        var invalidInputs = List.of( arrayType,
```

Comment:
Do we need to change `getWildcardType`'s behaviour for `ArrayType` or specification? `ArrayType` is a reference type, and `getWildcardType` is now specified to throw IAE for all reference types.

Reply:
Oops, disregard that previous comment. My bad, was sloppily reviewing this part.